tests/test_importers.py

```diff
@@ -157,6 +158,31 @@ def test_archiving_files(self):
         # Assert that there is no file in the input_files folder anymore
         assert len(os.listdir(moved_files_path)) == 0
 
```

Comment:
@BarisSari - IMHO this needs a bit more testing. We could do with:
* create a copy of the file, with a different name. Shouldn't load.
* create a copy of the file, with a different name, change size (delete character). Should load.
* create a copy of the file, with different name, same size, different hash (change character). Should load

Reply:
Okay, Ian. I'm implementing new tests

Reply:
New tests created:
https://github.com/debrief/pepys-import/pull/217/commits/2a36f69979c71ed6ca4ba346637fd9994f9cfd64